src/applications/financial-status-report/components/household/DependentAges.jsx

```diff
@@ -169,7 +169,7 @@ const DependentAges = ({ goToPath, isReviewMode = false }) => {
       ? 'form-review-panel-page-header vads-u-font-size--h5'
       : 'schemablock-title vads-u-margin-top--5';
   const text =
-    isReviewMode && !isEditing ? 'Review Dependants ages' : 'Dependents ages';
+    isReviewMode && !isEditing ? 'Dependants ages' : 'Dependents ages';
```

Comment:
So I think if we want the title to just be Dependent ages we can update this:

` <HeaderTag className={className}>{text}</HeaderTag>`
 to         
 `<HeaderTag className={className}>Dependent ages</HeaderTag>`
 
 and remove this ```const text =
    isReviewMode && !isEditing ? 'Dependants ages' : 'Dependents ages';```